lib/impure/db_sqlite.nim

```diff
@@ -10,9 +10,6 @@
 ## A higher level `SQLite`:idx: database wrapper. This interface
 ## is implemented for other databases too.
 ##
-## See also: `db_odbc <db_odbc.html>`_, `db_postgres <db_postgres.html>`_,
```

Comment:
This "See also" is nice, don't remove it please.

Reply:
It is still at the end of the introductory section, like it is done for other modules.